### PR TITLE
Adding FAQ

### PR DESCRIPTION
**Pull request by jkelvie**

### CODE REVIEW

File: unit-testing/faq.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+# **FAQs for Unit-Testing**
```
Comment (jkelvie) [inline unit-testing/faq.md]: The file needs header data, like we have in the other published docs, in order to be visible. It looks like this:
```
---
# Page settings
layout: default
keywords:
comments: false

# Hero section
title: Setting Up A Virtual Device

# Micro navigation
micro_nav: true
---
```

File: unit-testing/faq.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+Here you can find common questions on regards unit testing for voice apps.
```
Comment (jkelvie) [inline unit-testing/faq.md]: You add the page to main page by including it in the index.md. That contains menus and sub-menus for each section.

The FAQ for unit-testing should be the last item in the unit-testing section.

The end-to-end testing is not ready yet, so do not include that.

File: unit-testing/faq.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+For more information about testing voice apps, please read **[here](https://developer.amazon.com/blogs/alexa/post/e2f3d18c-13ca-4796-bc83-e8a196f20e57/building-engaging-alexa-skills-why-testing-and-automation-matter)**.
+
+### **Which tools does Bespoken offer to do unit testing for voice apps?**
+We have created a tool called **Virtual Alexa**, which simulates the Alexa Voice Service and creates JSON objects that are sent to your skill. This tool can be used to do unit testing programmatically and run the test suites with a unit testing framework (for example Mocha and Chai). But we also offer an *easier way* to perform unit testing by writing simple test scripts based on **YAML** syntax and running them with our **Skill Tester** tool. These scripts are executed *locally* using our Virtual Alexa simulator. Skill tester also displays the **code coverage** you reach based on your test scripts. To get started with Unit Test Scripts please **[read here](https://read.bespoken.io/unit-testing/getting-started)**.
```
Comment (jkelvie) [inline unit-testing/faq.md]: Do not refer to it as skill tester - people are just using bst, also known as the Bespoken CLI.

File: unit-testing/faq.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+### **What is code coverage?**
+In computer science, test coverage is a measure used to describe the degree to which the source code of a program is executed when a particular test suite runs. A program with high test coverage, measured as a percentage, has had more of its source code executed during testing, which suggests it has a lower chance of containing undetected software bugs compared to a program with low test coverage.
+
+*Bespoken Skill Tester for unit testing comes with Code Coverage out of the box*, you don't need to do anything to enable it, just check the results after executing your test scripts. Here you have a sample output. "% Lines" indicate the total number of lines that have been covered by the test script we are running.
```
Comment (jkelvie) [inline unit-testing/faq.md]: Bespoken Unit-Testing comes with

**Comment by ig-perez** [comment]
I've pushed a new version with the corrections

**Comment by ig-perez** [comment]
@jkelvie I've just added the first version of the FAQ for E2E and made some extra corrections to the one for Unit Testing

**Review by jkelvie — COMMENTED** [review]
Please take note - I made a number of formatting changes to make the site look better. Also, the images are not showing up on any of the FAQs, and it does not appear the image files are there.

Please place the images under the assets/images directory, and then reference them as `/assets/images/[IMAGE_NAME]`

I've also invited you as an administrator to netlify. When you make changes to this branch, the changes will be automatically visible on netlify. Look at the deploy history here: https://app.netlify.com/sites/zen-pike-2b04c4/overview

You can see the latest deploys and the links to the previews there to ensure it is working correctly.

**Comment by ig-perez** [comment]
@jkelvie I made some updates and the images are showing now.

**Pull request merged**